### PR TITLE
Add script covering case when VR.DeleteCommand gets UNSUPPORTED_RESOU…

### PR DESCRIPTION
**Pull request by vvvakulenko**
…RCE and UI.DeleteCommand gets WARNINGS from HMI
Script is part of TS: #666

### CODE REVIEW

**Comment by vvvakulenko** [comment]
@istoimenova Please review

File: test_scripts/API/DeleteCommand/ATF_DeleteCommand_UNSUP_RES_UI_to_DelCmd_WARNINGS_to_VR_DelCmd.lua [hunk 1]
```diff
@@ -0,0 +1,167 @@
+---------------------------------------------------------------------------------------------
```
Comment (vvvakulenko) [inline test_scripts/API/DeleteCommand/ATF_DeleteCommand_UNSUP_RES_UI_to_DelCmd_WARNINGS_to_VR_DelCmd.lua]: @istoimenova , script was updated in commit a8c398943f8b88c4af198b536c410f7a945416d0

Comment (vvvakulenko) [inline test_scripts/API/DeleteCommand/ATF_DeleteCommand_UNSUP_RES_UI_to_DelCmd_WARNINGS_to_VR_DelCmd.lua]: @istoimenova , script was updated in commit a8c398943f8b88c4af198b536c410f7a945416d0

File: test_scripts/API/DeleteCommand/ATF_DeleteCommand_UNSUP_RES_UI_to_DelCmd_WARNINGS_to_VR_DelCmd.lua [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+-- Requirement summary:
```
Comment (vvvakulenko) [inline test_scripts/API/DeleteCommand/ATF_DeleteCommand_UNSUP_RES_UI_to_DelCmd_WARNINGS_to_VR_DelCmd.lua]: @istoimenova , script was updated in commit a8c398943f8b88c4af198b536c410f7a945416d0

File: test_scripts/API/DeleteCommand/ATF_DeleteCommand_UNSUP_RES_UI_to_DelCmd_WARNINGS_to_VR_DelCmd.lua [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+-- SDL must send WARNINGS (success:true) to mobile app in case HMI respond WARNINGS at least to one component of RPC
```
Comment (vvvakulenko) [inline test_scripts/API/DeleteCommand/ATF_DeleteCommand_UNSUP_RES_UI_to_DelCmd_WARNINGS_to_VR_DelCmd.lua]: @istoimenova , script was updated in commit a8c398943f8b88c4af198b536c410f7a945416d0

File: test_scripts/API/DeleteCommand/024_ATF_UI_IsReady_no_resp_UI_DeleteCommand_UNSUP_RES.lua [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+
+  EXPECT_HMICALL("VR.DeleteCommand",
+  {
+    cmdID = 11,
+    type = "Command",
+    appID = self.applications[config.application1.registerAppInterfaceParams.appName]
+  })
+  :Do(function(_,data) self.hmiConnection:SendResponse(data.id, "VR.DeleteCommand", "WARNINGS", {}) end)
+
+  EXPECT_RESPONSE(cid, { success = true, resultCode = "UNSUPPORTED_RESOURCE", info = "unsupported resource"})
```
Comment (istoimenova) [inline test_scripts/API/DeleteCommand/024_ATF_UI_IsReady_no_resp_UI_DeleteCommand_UNSUP_RES.lua]: @vvvakulenko, you should wait for id of mobile request: cor_id_del_cmd

**Pull request merged**